sdk/keyvault/keyvault-keys/src/cryptographyClient.ts

```diff
@@ -142,7 +142,7 @@ export class CryptographyClient {
   /**
    * The ID of the key used to perform cryptographic operations for the client.
    */
-  get keyId(): string | undefined {
+  get keyID(): string | undefined {
```

Comment:
It makes me a little sad that this is `keyID` instead of `keyId`, but it does make it consistent with the rest of the interfaces in this package (if inconsistent with the rest of the keyvault packages)